source/_patterns/00-config/config.design-tokens.yml

```diff
@@ -41,6 +25,20 @@ gesso:
       gray-5: '#5b616b'
       gray-6: '#323a45'
       black: '#212121'
+    other:
+      yellow:
+        base: '#fad980'
+        light: '#fff1d2'
+        neon: '#ff0'
```

Comment:
This seems outside the naming conventions. Should this just be another color key `other.neon-yellow.base`?

Reply:
I'm good with that.  I did update the color.twig to handle things other than just base/dark/light just in case someone uses a different name, but that can just exist as a fallback.